plone/app/vocabularies/__init__.py

```diff
@@ -1,20 +1,20 @@
-# -*- coding: utf-8 -*-
 from plone.app.vocabularies.interfaces import IPermissiveVocabulary
 from plone.app.vocabularies.interfaces import ISlicableVocabulary
-from six.moves import urllib
 from zope.interface import directlyProvides
 from zope.interface import implementer
 from zope.schema.vocabulary import SimpleTerm
 from zope.schema.vocabulary import SimpleVocabulary
 
+import urllib
 
-_token_parse_py3 = getattr(urllib, 'parse', None)
-_token_parse_py27 = lambda token: urllib.unquote_plus(token).decode('utf8')
+
+_token_parse_py3 = getattr(urllib, "parse", None)
+_token_parse_py27 = lambda token: urllib.unquote_plus(token).decode("utf8")
 parse = _token_parse_py3.unquote if _token_parse_py3 else _token_parse_py27
```

Comment:
@jensens This could be simplified for Python 3 only

Reply:
Best as new PR then